lib/pure/strutils.nim

```diff
@@ -158,6 +161,14 @@ func isSpaceAscii*(c: char): bool {.rtl, extern: "nsuIsSpaceAsciiChar".} =
     doAssert isSpaceAscii('\t') == true
   return c in Whitespace
 
+func isPunctAscii*(c: char): bool {.rtl, extern: "nsuIsPunctAsciiChar".} =
```

Comment:
i don't think we need this proc, `c in Punctuation` seems good enough

Reply:
Ok. I'll remove it.